Makefile

```diff
@@ -19,20 +17,10 @@ run-all-tests: run-nunit-tests
 
 # $(call RUN_NUNIT_TEST,filename,log-lref?)
 define RUN_NUNIT_TEST
-	MONO_TRACE_LISTENER=Console.Out \
-	$(RUNTIME) \
-		$(NUNIT_CONSOLE) $(NUNIT_EXTRA) $(1) \
-		$(if $(RUN),-run:$(RUN)) \
-		--result="TestResult-$(basename $(notdir $(1))).xml" \
-		-output=bin/Test$(CONFIGURATION)/TestOutput-$(basename $(notdir $(1))).txt \
-	|| true ; \
-	if [ -f "bin/Test$(CONFIGURATION)/TestOutput-$(basename $(notdir $(1))).txt" ] ; then \
-		cat bin/Test$(CONFIGURATION)/TestOutput-$(basename $(notdir $(1))).txt ; \
-	fi
+	dotnet test $(1) -l "console;verbosity=detailed" \
+		-l "trx;LogFileName=bin/Test$(CONFIGURATION)/TestOutput-$(basename $(notdir $(1))).txt"
```

Comment:
Minor: the only thing to check is if this file is an xml file or plain text. We might want to keep the `.trx` extension if it is.

Reply:
Thanks good catch, updated.